o-fish-ios/Views/ReportFlow/Catch/UnitPickerWithButton.swift

```diff
@@ -10,19 +10,18 @@ import SwiftUI
 struct UnitPickerWithButton: View {
     typealias UnitSpecification = CatchViewModel.UnitSpecification
 
+    var selectButtonTitle: String = "Select"
+    var selectButtonClicked: ((UnitSpecification) -> Void)
+    @State private var unit: UnitSpecification = UnitSpecification.allCases[0]
```

Comment:
change to
@State private var unit = UnitSpecification.allCases.first ?? .notSelected